Dockerfile

```diff
@@ -68,6 +68,7 @@ ARG REACT_APP_TUNNISTAMO_SCOPE_BERTHS
 ARG REACT_APP_TUNNISTAMO_SCOPE_PROFILE
 ARG REACT_APP_TUNNISTAMO_URI
 ARG REACT_APP_BERTH_API_URL
+ARG REACT_APP_LOCAL_ENV_URL
```

Comment:
You need to add `ENV REACT_APP_LOCAL_ENV_URL $REACT_APP_LOCAL_ENV_URL` under this.
This could also be higher in this ARG list near `REACT_APP_ENV`

Reply:
Browser tests are configured to read its environment variables from `.env.development.local`, therefore, I moved this variable there, and add it to GH Secrets.